astropy/table/tests/test_table.py

```diff
@@ -2080,13 +2081,21 @@ def test_simple(self):
 
         d = t.to_pandas()
 
+        if minversion("pandas", "3.0.0.dev"):
```

Comment:
I think for things like this, we do a `PANDAS_LT_3_0_0` variable assignment so we could easily grep, though that is not set in stone.